docs/Installation.md

```diff
@@ -116,9 +124,9 @@ Check the [declaration of prerequisites] to make sure your preferred combination
 of operating system version and database engine version is supported.
 
 The installation instructions below assumes that this is a new installation.
-If you upgrade and want to keep the database, go to [section 7][Upgrade
-database] first. If you instead want to start from afresh, then go to [appendix
-C][Cleaning database] and remove the old database first.
+If you upgrade and want to keep the database, go to section 
+["Upgrade Zonemaster database"] first. If you instead want to start from afresh,

```

Comment:
Would it be possible to remove the quote from the link text?

Reply:
Yes, I'll do that, but they were there on purpose.

Reply:
Oh I didn't know that. Could you elaborate?

Reply:
> but they were there on purpose.

I was curious to know more on the purpose mentioned.

Reply:
> > but they were there on purpose.
> 
> I was curious to know more on the purpose mentioned.

Better appearance, easier to read.

Reply:
TBH I haven't checked how it was rendered. I just found it unusual to include the quotes within the link id.